src/cascadia/TerminalSettingsModel/SettingsUtils.h

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Curious, why is this a separate file? When, in the future, should I add something here instead of, say, Terminal Settings?

Reply:
It's just a utils header - there was no good place for shared code in the settings binary. It didn't compile in the WinRTUtils binary (cause dependencies on Windows.UI.Xaml or something), so I just stuck it here

Reply:
I feel like this might be good opportunity to rename `FileUtils` to just `Utils` and put it there.